src/main/java/sbt_inc/SbtLogger.java

```diff
@@ -30,14 +30,19 @@ public void success(Function0<String> message) {
 
     @Override
     public void log(Enumeration.Value level, Function0<String> message) {
+        String s = message.apply();
+        String prefix = "[" + level.toString().toLowerCase() + "] ";
+        if (s.substring(0, prefix.length()).toLowerCase().equals(prefix)) {
```

Comment:
Please remove toLowerCase and use [String#regionMatches](https://docs.oracle.com/javase/7/docs/api/java/lang/String.html#regionMatches(boolean,%20int,%20java.lang.String,%20int,%20int)).

Reply:
I didn’t know about String#regionMatches.  That’s great!